plugin/pkg/scheduler/scheduler.go

```diff
@@ -89,6 +92,12 @@ func (s *Scheduler) Run() {
 
 func (s *Scheduler) scheduleOne() {
 	pod := s.config.NextPod()
+
+	// If the assumed pods store contains this pod there is a conflict,
+	// since the store can't contain a pod without a host, and NextPod
+	// only gives us pods without hosts.
+	s.config.Modeler.ForgetAssumedPod(pod)
```

Comment:
I'd love to take credit for whatever this change that was made is, but I'm pretty sure I didn't have anything to do with it.